CMakeLists.txt

```diff
@@ -249,9 +257,6 @@ else()
   message(STATUS "eventdisplay will not be built, because ROOT has no opengl support.")
 endif()
 
-if(BUILD_MBS)
-  add_subdirectory(MbsAPI)
-endif()
```

Comment:
Dennis suggested to have all "components" (exported targets) in sibling directories.
So please just keep MbsAPI in its current directory?

Reply:
Okay. Will do so.